source/docs/causal_profiling.md

```diff
@@ -37,7 +39,7 @@ void bar()
 
 int main()
 {
-    auto _threads = { std::thread{ foo },
+    std::thread _threads[] = { std::thread{ foo },
```

Comment:
Ah yeah, I don’t have a problem with this but I meant for this to be pseudo code so I wanted to avoid any distracting complexity